include/oneapi/dpl/pstl/hetero/dpcpp/parallel_backend_sycl.h

```diff
@@ -706,6 +696,7 @@ __parallel_transform_scan_single_group(oneapi::dpl::__internal::__device_backend
                         /* _IsFullGroup= */ ::std::false_type, _Inclusive, _CustomName>>>()(
                     ::std::forward<_ExecutionPolicy>(__exec), std::forward<_InRng>(__in_rng),
                     std::forward<_OutRng>(__out_rng), __n, __init, __binary_op, __unary_op);
+            return __future(__event, __dummy_result_and_scratch);
```

Comment:
```suggestion
            return __future(__event, _TempStorage{__exec, 0});
```

Reply:
In this case `_TempStorage __dummy_result_and_scratch{__exec, 0};` will not required anymore.

Reply:
I disagree with this suggestion, because while we do save a line of code, we lose the ability to name it `__dummy_result_and_scratch`. 

I think this is descriptive, and helpful to show that this is unused to the future reader.